packages/@biomejs/backend-jsonrpc/tests/transport.test.mjs

```diff
@@ -1,5 +1,4 @@
 import { describe, expect, it, vi } from "vitest";
-
```

Comment:
You mentioned "Adding blank line between custom group is not supported", but does that mean the import organiser actively removes them if they exist? Maybe you removed this manually, but if this line was removed by the import organiser, I do think that's a serious regression that users will surely complain about.

Reply:
> but does that mean the import organiser actively removes them if they exist?

Yes, it is an intended behavior.

Some IDE adds a blank line before adding new imports. By ignoring the blank line we are able to order all imports.
Once you sort all imports it doesn't make sense of keeping the blank line.

What do you suggest? Keeping blank lines even if we move the imports?

Reply:
> And it means that groups that are intentionally separated will now be forcibly grouped together. I expect anyone that relied on/valued the previous behaviour would see that as a regression.

I changed the behavior because some users complained about that...
Now you have to add a detached comment to ensure a chunk break:

```js
import { B } from "b"

// Detached

import { A } from "a"
```

Reply:
Maybe it needs to be an option already then? I can understand some users will like the new behaviour and I don’t think it’s bad, but I do think it will upset others…